src-docs/src/views/call_out/playground.js

```diff
@@ -0,0 +1,57 @@
+import { PropTypes } from 'react-view';
+import { EuiCallOut, EuiText } from '../../../../src/components/';
+import {
+  propUtilityForPlayground,
+  mapOptions,
+} from '../../services/playground';
+import { iconTypes } from '../icon/icons';
+
+export default () => {
+  const docgenInfo = Array.isArray(EuiCallOut.__docgenInfo)
+    ? EuiCallOut.__docgenInfo[0]
+    : EuiCallOut.__docgenInfo;
+  const propsToUse = propUtilityForPlayground(docgenInfo.props);
+  const options = mapOptions(iconTypes);
+
+  propsToUse.iconType = {
+    ...propsToUse.iconType,
+    value: undefined,
+    type: PropTypes.String,
+    custom: {
+      ...propsToUse.iconType.custom,
+      validator: val => options[val],
+    },
+  };
+
+  propsToUse.title = {
+    ...propsToUse.title,
+    value: 'Check it out',
+    type: PropTypes.String,
+  };
+
+  propsToUse.children = {
+    ...propsToUse.children,
+    value: `<p>
+   Any content inside of <strong>EuiCallOut</strong> will appear here.
+  </p>`,
+    type: PropTypes.ReactNode,
+    description: 'Visible label.',
```

Comment:
Oh actually one thing that I didn't realize was coming from this config, but the description isn't quite right.
```suggestion
    description: 'Content to display inside the callout below the title',
```